x-pack/plugin/ent-search/src/main/java/module-info.java

```diff
@@ -6,10 +6,12 @@
  */
 
 module org.elasticsearch.application {
+    requires org.apache.logging.log4j;
```

Comment:
Most of the time this is not a required dependency.

You may want to replace occurrences of 

```java
import org.apache.logging.log4j.LogManager;
import org.apache.logging.log4j.Logger;
```

with

```java
import org.elasticsearch.logging.LogManager;
import org.elasticsearch.logging.Logger;
```

We should refrain to add dependencies unless it is really required.

Reply:
Thanks, that's a good callout. I was able to remove the `org.apache.logging.log4j` dependency but not the `org.slf4j` dependency as that's required by Jackson.

Reply:
We don't need the Jackson dependency in this PR so let's use the `org.elasticsearch.logging` import for now and we can discuss the other needs in the follow up PR?